### PR TITLE
Add notes to migration guide about the config file

### PR DESCRIPTION
**Pull request by JasoonS**

### CODE REVIEW

File: docs/HyperIndex/migration-guide-v1-v2.md [hunk 1]
```diff
@@ -42,6 +42,7 @@ V2 of HyperIndex is about streamlining the process of starting an indexer and op
 ### Configuration
 
 - There is no async-mode anymore, so you can remove `isAsync: true` from each of the events in your `config.yaml`.
+- This is no more 'required_entities' in the config file. This includes subfields such as `label` and `arrayLabels`.
```
Comment (DZakh) [inline docs/HyperIndex/migration-guide-v1-v2.md:45]:
```suggestion
- There is no more 'required_entities' in the config file. This includes sub-fields such as `label` and `arrayLabels`.
```

File: docs/HyperIndex/migration-guide-v1-v2.md [hunk 2]
```diff
@@ -257,6 +258,34 @@ Greeter.Event1.handlerWithLoader({
...
+          - name: Greetings
+            arrayLabels: previousGreetings
+```
+
+After:
+```yaml
+contracts:
+  - name: Greeter
+    handler: src/EventHandlers.ts
+    sameRandomFieldThatIsntPartOfSchema: true
```
Comment (DZakh) [inline docs/HyperIndex/migration-guide-v1-v2.md:284]:
```suggestion
```

**Review by DZakh — APPROVED** [review]
Nice

**Review by JonoPrest — APPROVED** [review]
Thanks Jason!

**Pull request merged**